app/components/bulk_upload_error_row_component.html.erb

```diff
@@ -18,16 +21,49 @@
         <% end %>
 
         <%= table.with_body do |body| %>
-          <% bulk_upload_errors.each do |error| %>
+          <% critical_errors.each do |error| %>
             <% body.with_row do |row| %>
-              <% row.with_cell(header: true, text: error.cell) %>
-              <% row.with_cell(text: question_for_field(error.field)) %>
-              <% row.with_cell(text: error.error) %>
-              <% row.with_cell(text: error.field.humanize) %>
+                <% row.with_cell(text: error.cell) %>
+                <% row.with_cell(text: question_for_field(error.field)) %>
+                <% row.with_cell(text: error.error, html_attributes: {class: "govuk-!-font-weight-bold"}) %>
+                <% row.with_cell(text: error.field.humanize) %>
             <% end %>
           <% end %>
         <% end %>
       <% end %>
     <% end %>
+
+    <% if potential_errors.any? %>
+      <h2 class="govuk-heading-m">Potential errors</h2>
+      <p class="govuk-body">The following groups of cells might have conflicting data. Check the answers and fix any incorrect data.<br><br>If the answers are correct, fix the critical errors and reupload the file. You'll need to confirm that the following data is correct when the file only contains potential errors.</p>
+        <%= govuk_table do |table| %>
+          <%= table.with_head do |head| %>
+            <% head.with_row do |row| %>
+              <% row.with_cell(header: true, text: "Cell") %>
+              <% row.with_cell(header: true, text: "Question") %>
+              <% row.with_cell(header: true, text: "Potential error") %>
+              <% row.with_cell(header: true, text: "Specification") %>
+            <% end %>
+          <% end %>
+
+          <%= table.with_body do |body| %>
+            <% potential_errors.group_by(&:error).each do |error_message, errors| %>
+              <% errors.each_with_index do |error, index| %>
+                <% row_class = "grouped-rows" %>
+                <% row_class += " first-row" if index.zero? %>
+                <% row_class += " last-row" if index == errors.size - 1 %>
+                <% body.with_row(html_attributes: { class: row_class }) do |row| %>
```

Comment:
I'm not entirely sure why we need this, as in, I can understand why we want to remove the border underneath the last row, but I'm not sure why we're removing it in other places. Looking at this it also doesn't seem like it's doing what I'd expect https://review.submit-social-housing-data.levellingup.gov.uk/2545/lettings-logs/bulk-upload-results/1

Reply:
It's not to remove the border on the last row (that can be done but did not to keep things consistent). 

For a given error message there might be multiple rows responsible, each of those are grouped together and the error message is only displayed once with it spread across the rows using `rowspan`. This lines of code are to outline the first row and last row of the group, remove the borders of each of the interior rows and only keep the top and bottom border of the whole group. Now though that the error message is no longer a header `<th>` and now `<td>` I will need to adjust the styling.

Reply:
Ahh, okay! I see now that you're grouping them! Any thoughts on the styling in the url I linked? Not sure if you can access it, so here's a screenshot:
<img width="882" alt="image" src="https://github.com/user-attachments/assets/537151a5-0052-4564-a473-214cd398dd5d">

Reply:
That is the error message now being <td> causing this, fixing it now.